### PR TITLE
Use Go 1.13 error wrapping

### PR DESCRIPTION
**Pull request by cloudlena**
As reported by https://github.com/Djarvur/go-err113
Fixes https://github.com/tormoder/fit/issues/40

### CODE REVIEW

File: cmd/fitgen/internal/profile/generator_test.go [hunk 1]
```diff
@@ -23,6 +24,10 @@ const (
 	currentSuffix = ".current"
 )
 
+var (
+	errInvalidFirstLine = errors.New("invalid first line")
```
Comment (subtlepseudonym) [inline cmd/fitgen/internal/profile/generator_test.go:28]: This might be overkill

**Review by subtlepseudonym — APPROVED** [review]
One non-blocking nitpick, otherwise LGTM

File: cmd/fitgen/internal/fitstringer/errors.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package fitstringer
```
Comment (tormoder) [inline cmd/fitgen/internal/fitstringer/errors.go:1]: This is manually "vendored" code (https://pkg.go.dev/golang.org/x/tools/cmd/stringer) with a few small changes for the `fitgen`. We should do no changes here to keep the diff with upstream as small as possible.

File: cmd/fitgen/internal/profile/errors.go [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package profile
```
Comment (tormoder) [inline cmd/fitgen/internal/profile/errors.go:1]: We don't need to introduce these types until we need to do anything specific based on the type. Currently the only user, `cmd/fitgen`. does not.

Wrapping. `%w`, could potentially stay.

**Comment by cloudlena** [comment]
Thanks, @tormoder. Should be all good now :)

**Comment by subtlepseudonym** [comment]
Changes look good! Although I'll miss the exported error variables.

I would gently suggest that you avoid force-pushing so many times to non-draft pull-request branches. It's not a big deal with smaller change sets like this one, but deleting history makes reviewing more difficult.

**Comment by tormoder** [comment]
Thanks!

**Pull request merged**